SparkPost/2web2ui
# FE-1207 - Remove the feature flag and remove the sending domain and tracking domain pages

FE-1207 - Remove the feature flag and remove the sending domain and tracking domain pages from nav in hibana

### What Changed
 -  Removed the feature flag - allow_domains_v2
 - Removed Sending and Tracking Domain from navigation in Hibana
 - Removed the routes to sending domain and tracking domain in hibana
 - OG theme still has the old sending and tracking domain pages intact

### How To Test
 - Check on a account with allow_domain_v2 not enabled, domains page is visible in the nav
 - Check in OG version the sending and tracking domain pages are visible

### To Do
